sql/promscale--0.0.0.sql

```diff
@@ -448,7 +468,9 @@ BEGIN
         ('011-tables-exemplar.sql'        , '0.0.0'),
         ('012-tracing.sql'                , '0.0.0'),
         ('013-tracing-well-known-tags.sql', '0.0.0'),
-        ('014-telemetry.sql'              , '0.0.0')
+        ('014-telemetry.sql'              , '0.0.0'),
+        ('015-tracing-compression.sql'    , '0.0.0'),
+        ('015-tracing-large-tags-fix.sql' , '0.0.0')
```

Comment:
These need to be removed. Adding them here says that these migrations have already been applied, however, these migrations show up in `0.5.0`. So, they need to be applied both in the upgrade and fresh install paths. Removing them from this list will cause them to be applied in both cases.

Reply:
Ok that was not clear. I hope we can make code comment more meaningful. 
eg. what we have now `-- Bring migrations table up to speed` doesn't look descriptive :)